### DISCUSSION TITLE
multiple.intoto.jsonl

### DISCUSSION THREAD

**Discussion by un99known99** [PrivateBin/PrivateBin#1247] [Q&A] [answered]
### Describe the problem/question

What is the file "multiple.intoto.jsonl" and why/how/where should it be used?

### Did you use the FAQ section?

- [X] Yes, I have read [the FAQ](https://github.com/PrivateBin/PrivateBin/wiki/FAQ) and I found no solution/answer there.

### What you did?

1.
2.
...

### Issue reproducibility

No, I cannot reproduce it on https://privatebin.net.

**Comment by r4sas** [comment, marked as answer]
Please look here: https://github.com/PrivateBin/PrivateBin/issues/1169#issuecomment-1857375199

Add: One more: https://github.com/PrivateBin/PrivateBin/blob/master/doc/Release.md

**Comment by un99known99** [comment]
@r4sas 
thx for the answer, but I am sorry, I still didnt get why this is needed out of the given infos - anything which needed to be installed in addition for what reason?

**Reply by rugk** [reply]
You don'r need it. If you want to verify the release though, you can use the SLSA verifier to verify it with the file, like this: https://github.com/slsa-framework/slsa-verifier?tab=readme-ov-file#verification-for-github-builders
